smartseq2_single_sample/check_reference/tasks/featurecounts.wdl

```diff
@@ -0,0 +1,47 @@
+task FeatureCountsUniqueMapping {
+  File aligned_bam
+  File gtf
+  String fc_out
+  
+  command {
+    featureCounts -T 4  -s 0 -t exon -g gene_id -p -B -C -a "${gtf}" -o "${fc_out}.gene.unq.counts.txt" "${aligned_bam}"
+    featureCounts -T 4  -s 0 -t exon -g transcript_id -p -B -C -a "${gtf}" -o "${fc_out}.transcript.unq.counts.txt" "${aligned_bam}"
+    featureCounts -T 4  -s 0 -t exon -g exon_id -p -B -C -a "${gtf}" -o "${fc_out}.exon.unq.counts.txt" "${aligned_bam}"
+  }
+  runtime {
+    docker:"humancellatlas/star_dev:v1"
```

Comment:
@jishuxu This should also be changed to "humancellatlas/star_dev:2.5.3a"

Reply:
Just changed